packages/idyll-components/src/switch.js

```diff
@@ -0,0 +1,34 @@
+const React = require('react');
+
+class Switch extends React.Component {
+  render() {
+    const { idyll, hasError, updateProps, value, ...props } = this.props;
+
+    if (props.children) {
+      const filterFn = value => ({ props }) =>
+        props.children[0].props.test === value;
+      const matchCase = filterFn(value);
+      const matchDefault = filterFn('Default');
```

Comment:
This is an edge case, but does this mean we couldn't match for the value `Default`? I wonder if its worth including a `[Default /]` component or `[Switch.Default /]` component to cover this functionality.

Reply:
I think having a default component is better than passing "default" as a string. So I have created a default component.

Also, I am using only one filter function to find both the matched case and default case. And used array destructuring to extract the matched case and default case. This, however, restricts users to always write the default case at the end. I am not clear whether having this restriction is good or bad (as this restriction is not there in javascript) lmk what you think about it.

Reply:
Having the `default` last seems like a reasonable case but I think there a bug in the logic in the case that a default isn't. I've updated to instead run two filters